### ISSUE TITLE
"Reflection" gets deleted when you complete an already completed and published solution

### ISSUE THREAD

**Issue by mgmatola** [exercism/v2-feedback#181]
Title says it all.

**Comment by iHiD** [comment]
@mgmatola Can you explain how you completed a completed solution pls?

**Comment by mgmatola** [comment]
https://docs.google.com/document/d/1tWRWxeFdDHBaDRV-3bsLdWIoXJ1xknM8p37-DgH8wsA/edit?usp=sharing

**Comment by mgmatola** [comment]
And I suspect I might have figured out the valid use case for completing a completed solution.

I have a solution that the mentor approved and I published. I still wanted to tweak it based on a comment made by the mentor. I wrote another solution. When I used the CLI to submit, I got a 505 error, but it still submitted. Both old (#1) and new (#2) solutions showed up in the mentor conversation, but only solution #1 appeared as my published solution. I went ahead and completed solution #2. Now, solution #2 has completed replaced solution #1 as the published solution. That is, I now have only a single published solution -- #2. But back on the mentor conversation page I can still access both #1 and #2.

I suspect some (most? all?) of this behavior is intentional? I have to admit that I'm a bit mystified as to the workflow. Is any of this a consequence of my being both a mentor and student?
